app/views/categories/show.html.erb

```diff
@@ -11,16 +11,18 @@
 <% end %>
 
 <% post_count = @posts.count %>
-<div class="has-color-tertiary-500 category-meta" title="<%= post_count %>">
-  <div>
+<div class="has-color-tertiary-500 category-meta">
+  <div title="<%= post_count %> posts">
```

Comment:
This div and the 2 that follow it (the post count, the Edit Category link, and the Subscribe button) need to be all contained within a single div. This time an extra layer of containment really is the answer. Without it, the styling of the next div up causes them to spread out to fill the available horizontal space (so the further you zoom out, the further they move apart from each other, rather than staying all on the left). I hadn't anticipated this when I suggested this would be an easy issue, but this will fix the spreading out problem.

Reply:
When I made these changes, the buttons all moved onto a new line.  I'm going to work through this again and see if I can figure out what's going on, but this didn't work on a first pass.